src/Sites/twitch.tv/Components/MessageRenderer.tsx

```diff
@@ -29,7 +29,7 @@ export class MessageRenderer {
 			c.remove();
 		});
 
-		const container = el.querySelector('.chat-line__no-background') ?? el.querySelector(Twitch.Selectors.ChatMessageContainer) ?? el;
+		const container = el.querySelector('.chat-line__no-background, .video-chat__message') ?? el.querySelector(Twitch.Selectors.ChatMessageContainer) ?? el;
```

Comment:
Doing an "and" query here might be unnecessary, perhaps detect if this is live or video chat and use a ternary operator, or similar?